packages/generator-one-app-module/package-lock.json

```diff
@@ -0,0 +1,2842 @@
+{
+	"name": "generator-one-app-module",
```

Comment:
You should re generate this file since the package name has changed in package.json.